src/config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "dialect": "postgres",
+    "host": "127.0.0.1",
+    "database": "chatdatabase",
+    "username": "maty",
```

Comment:
Even though this is a sample project, you shouldn't use your name as the db's username. We usually use the same name for db name and username. In your case `chatdatabase`.

Reply:
Fixed